### ISSUE TITLE
intermittent "rsync: Command not found"

### ISSUE THREAD

**Issue by niklasf** [vmactions/freebsd-vm#13]
Using default settings only, occasionally the build process will fail before even running the custom command. It looks like somehow `rsync` is not getting installed.

`10 Feb 17:20:30 ntpdate[802]: no server suitable for synchronization found` might also be notable.

```
Wed Feb 10 17:20:17 UTC 2021

FreeBSD/amd64 (freebsd) (ttyv0)

logi

 

OK
Sync FreeBSD time
exec ssh: ntpdate -b pool.ntp.org
/usr/bin/ssh -t freebsd
Pseudo-terminal will not be allocated because stdin is not a terminal.
Warning: Permanently added '[localhost]:2222' (ECDSA) to the list of known hosts.
Warning: no access to tty (Bad file descriptor).
Thus no job control in this shell.
10 Feb 17:20:30 ntpdate[802]: no server suitable for synchronization found
Error: The process '/usr/bin/ssh' failed with exit code 1
envs:
run: pkg install -y gmake curl && cd src && env CXXFLAGS=-DNNUE_EMBEDDING_OFF gmake ARCH=x86-64-sse3-popcnt COMP=clang profile-build && strip stockfish
/usr/bin/ssh freebsd sh -c 'cd $GITHUB_WORKSPACE && exec $SHELL'
cd: /Users/runner/work/Stockfish/Stockfish: No such file or directory
Error: The process '/usr/bin/ssh' failed with exit code 2
get back by rsync
/usr/bin/rsync -uvzrtopg freebsd:work/ /Users/runner/work
rsync: Command not found.
rsync: connection unexpectedly closed (0 bytes received so far) [receiver]
rsync error: error in rsync protocol data stream (code 12) at /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-54.120.1/rsync/io.c(453) [receiver=2.6.9]
Error: The process '/usr/bin/rsync' failed with exit code 12
```

Example run: https://github.com/niklasf/Stockfish/runs/1866551256?check_suite_focus=true

**Comment by niklasf** [comment]
FWIW the workflow has not failed since I disabled the ntpdate setup step, so maybe it's due to a firewall intermittently blocking ntp?

```diff
diff --git a/index.js b/index.js
index de8e27a..ae732e3 100644
--- a/index.js
+++ b/index.js
@@ -146,7 +146,7 @@ async function setup(nat, mem) {
     let loginTag = "FreeBSD/amd64 (freebsd) (ttyv0)";
     await waitFor(vmName, loginTag);

-    await execSSH("ntpdate -b pool.ntp.org", "Sync FreeBSD time");
+    //await execSSH("ntpdate -b pool.ntp.org", "Sync FreeBSD time");

     let cmd1 = "mkdir -p /Users/runner/work && ln -s /Users/runner/work/  work";
     await execSSH(cmd1, "Setting up VM");
```

**Comment by niklasf** [comment]
most likely now fixed via 457af7345642e154a79d219971a2d4a7c7fe2118
